packages/devtools-node/src/fs.ts

```diff
@@ -25,9 +25,13 @@ export function getDirPath(path: PathInput): string {
 /**
  * Create a Composite from a GraphQL schema path.
  */
-export async function createComposite(ceramic: CeramicClient, path: PathInput): Promise<Composite> {
+export async function createComposite(
+  ceramic: CeramicClient,
+  path: PathInput,
+  deploy: boolean,
```

Comment:
That's a breaking API change, please set a default value or bump to v0.7 with docs updates.